Woof/WoofTests/Helper/AppButton/AppButtonDescriptionTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+final class AppButtonDescriptionTests: XCTestCase {
+    func testAppButtonDescriptionReturnsExpectedString() {
+        // Given
+        let cancel = "Cancel"
+        let save = "Save"
+        let edit = "Edit"
+        let tryAgain = "Try Again"
+        let logout = "Logout"
+        let `continue` = "Continue"
+
+        // When // Then
+        XCTAssertEqual(AppButton.cancel.rawValue, cancel)
```

Comment:
Could we simplify it? We do not have to create  constants and call them only once as we can compare values in asserts. WDY?
```suggestion
        // Given
        let cancel = "Cancel"
        let save = "Save"
        let edit = "Edit"
        let tryAgain = "Try Again"
        let logout = "Logout"
        let `continue` = "Continue"

        // When // Then
        XCTAssertEqual(AppButton.cancel.rawValue, "Cancel")
```

Reply:
Fixed with 03725e9